### ISSUE TITLE
Unexpected error message on case-sensitive file system when file is included in program both via relative and absolute names

### ISSUE THREAD

**Issue by vladima** [Microsoft/TypeScript#5274]
```ts
// a.ts
<reference path="b.ts"/>
```
```ts
// b.ts
var x
```

```
vladima@vladima-OptiPlex-990:~/Sources/TypeScript$ node built/local/tsc.js a.ts $(pwd)/b.ts
error TS1149: File name '/home/vladima/Sources/TypeScript/b.ts' differs from already included file name 'b.ts' only in casing

```

**Comment by vladima** [comment]
fixed in #5275
